strawberryfields/apps/similarity.py

```diff
@@ -306,33 +320,148 @@ def event_cardinality(photon_number: int, max_count_per_mode: int, modes: int) -
     return cardinality
 
 
-def prob_orbit_mc(
-    graph: nx.Graph, orbit: list, n_mean: float = 5, samples: int = 1000, loss: float = 0.0
+def _prob_orbit_exact(
+    graph: nx.Graph,
+    orbit: list,
+    n_mean: float = 5,
+    loss: float = 0.0
 ) -> float:

```

Comment:
When running `black`, are you doing it with a line width of `100`?

`black -l 100 strawberryfields/apps/similarity.py` should be the right command.

Reply:
Thanks. I was not running with this line width. I have implemented it now. :)